obspy/signal/spectral_estimation.py

```diff
@@ -1824,6 +1832,31 @@ def plot(self, filename=None, show_coverage=True, show_histogram=True,
                 else:
                     xdata = periods
                 ax.plot(xdata, noise_model, '0.4', linewidth=2, zorder=10)
+        
+        if show_earthquakes:
+            earthquakes=get_earthquakes()
+            for eqk in earthquakes:
+                # Eq.1 Cauzzi & Clinton (2013)
```

Comment:
please also include this second reference in docstring and the bibliography.

Reply:
OK.